homeassistant/components/tibber/strings.json

```diff
@@ -1,4 +1,89 @@
 {
+  "entity": {
+    "sensor": {
+      "electricity_price": {
+        "name": "Electricity price"
+      },
+      "month_cost": {
+        "name": "Monthly cost"
+      },
+      "peak_hour": {
+        "name": "Monthly peak hour consumption"
+      },
+      "peak_hour_time": {
+        "name": "Time of max hour consumption"
+      },
+      "month_cons": {
+        "name": "Monthly net consumption"
+      },
+      "average_power": {
+        "name": "average power"
```

Comment:
These need to start with a capital letter?

Reply:
Oh yea indeed, just copied that